esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -1563,7 +1558,32 @@ bool CWsWorkunitsEx::onWUInfo(IEspContext &context, IEspWUInfoRequest &req, IEsp
             resp.setCanCompile(notEmpty(context.queryUserId()));
             if (version > 1.24 && notEmpty(req.getThorSlaveIP()))
                 resp.setThorSlaveIP(req.getThorSlaveIP());
-            resp.setSecMethod(authMethod.get());
+
+            ISecManager* secmgr = context.querySecManager();
+            if (!secmgr)
+                resp.setSecMethod(NULL);
+            else
+            {
+                switch (secmgr->querySecMgrType())
```

Comment:
What happens when we add another? Will we remember to change this?

Giving ISecManager a method querySecMgrTypeName, or having a static method in a shared security dll (is there one?) that maps types to strings would be a better approach.